client/apps/edit/components/admin/index.coffee

```diff
@@ -308,17 +308,21 @@ module.exports = class EditAdmin extends Backbone.View
       featuredListTemplate list: @article.featuredList('Artworks')
     )
 
+  renderInputs: (src) ->
+    $('.edit-email-large-image-url input').val(
+      crop(src, { width: 1280, height: 960 } ))
```

Comment:
Ah, it would be a huge win to get rid of Embedly... ST doesn't have any image processing except for an Image Library but there isn't a programmatic way to do it except to click and upload. Are you thinking of cropping in-house like we do with `resize`? That could easily be swapped out here if we add it in the future.